tests/ui/const-generics/adt_const_params/unsized_field-1.stderr

```diff
@@ -6,6 +6,12 @@ LL | #[derive(ConstParamTy, Eq, PartialEq)]
 LL |
 LL | struct A([u8]);
    |          ---- this field does not implement `ConstParamTy_`
+   |
+note: the `ConstParamTy_` impl for `[u8]` requires that `unstable feature: `unsized_const_params``
```

Comment:
we should probably make the diagnostic for unsatisfied `UnstableFeature` clauses be worded a bit nicer. different PR though